src/category_theory/triangulated/basic.lean

```diff
@@ -72,68 +66,50 @@ In other words, we have a commutative diagram:
 
 See https://stacks.math.columbia.edu/tag/0144.
 -/
-structure triangle_morphism (T₁ : triangle C) (T₂ : triangle C):=
+@[ext]
+structure triangle_morphism (T₁ : triangle C) (T₂ : triangle C) :=
 (trimor1 : T₁.obj1 ⟶ T₂.obj1)
 (trimor2 : T₁.obj2 ⟶ T₂.obj2)
 (trimor3 : T₁.obj3 ⟶ T₂.obj3)
-(comm1: T₁.mor1 ≫ trimor2 = trimor1 ≫ T₂.mor1)
-(comm2: T₁.mor2 ≫ trimor3 = trimor2 ≫ T₂.mor2)
-(comm3: T₁.mor3 ≫ trimor1⟦1⟧' = trimor3 ≫ T₂.mor3)
```

Comment:
In fact, mathlib style probably suggests we should replace all the numerals with subscript numerals, but I won't insist. :-)